lib/compress/zstdmt_compress.c

```diff
@@ -460,6 +464,8 @@ struct ZSTDMT_CCtx_s {
     unsigned frameEnded;
     unsigned allJobsCompleted;
     unsigned long long frameContentSize;
+    unsigned long long consumed;
+    unsigned long long produced;
```

Comment:
`U64`

Reply:
These variables are using exactly the same type as `ZSTD_frameProgression` members.
`U64` and `unsigned long long` are supposed to be similar in most situations,
but just in case they aren't, I prefer to preserve type continuity.